src/dev-app/index.html

```diff
@@ -25,13 +25,30 @@
 <body>
   <dev-app>Loading...</dev-app>
 
+  <iframe id="google-maps-api-key"
+          src="google-maps-api-key.txt"
+          style="display:none;"
+          onload="loadGoogleMapsScript()"></iframe>
+
   <script src="core-js-bundle/index.js"></script>
   <script src="zone.js/dist/zone.js"></script>
   <script src="systemjs/dist/system.js"></script>
   <script src="system-config.js"></script>
   <script src="https://www.youtube.com/iframe_api"></script>
-  <script src="https://maps.googleapis.com/maps/api/js"></script>
   <script src="https://unpkg.com/@googlemaps/markerclustererplus/dist/index.min.js"></script>
+  <script>
+    function loadGoogleMapsScript() {
+      var iframe = document.getElementById('google-maps-api-key');
+      var googleMapsScript = document.createElement('script');
+      var googleMapsApiKey = iframe.contentDocument.body.textContent;
```

Comment:
Why do we need the API key now? Is it not possible to use the service without one?

Reply:
Yes, the DirectionsService route method will fail without a properly setup API key. I've set it up so that everything should still work in the demo without the API key except for the Directions request.